templates/kubernetes/docs/contact.md

```diff
@@ -0,0 +1,48 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Contacts"
+  description: How to contact the CDK team
+keywords: help, contact, support
+tags: [contact]
+sidebar: k8smain-sidebar
+permalink: contact.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The team behind the **Charmed Distribution of Kubernetes<sup>&reg;</sup>** are
+keen to help and listen to your feedback. There are a variety of ways to get in touch
+with suggestions, problems or just to get involved.
+
+## IRC
+
+Join us on freenode.net in **#cdk8s**.
+
+## Slack
+
+Find us in **#cdk** on the [Kubernetes slack][slack].
+
+## Bugs
+
+**CDK** bugs  are tracked in [launchpad][lp].
+
+## Documentation
+
+Visit the [documentation site][docs] for issues or comments about this documentation.
```

Comment:
Do you think documentation repository would be a better name here? Wouldn't the user already believe they were on the documentation site?

Reply:
Ok, I'll fix this